.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+# See GitHub's docs for more information on this file:
+# https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
```

Comment:
do we need to add this, this repo doesn't have any Github actions?

Reply:
I agree it doesn't make sense, but having this file is a requirement for ops level 🙃

Reply:
What dependencies are you looking to maintain with this?

Reply:
Nothing, just ops level doesn't like this repo because it doesn't have this file and I don't see any "this repo doesn't need this file option"

Reply:
Can we add a comment that this does nothing and here to satisfy OpsLevel?

So that when OpsLevel is gone we aren't left with random files wondering "why we need this".

Reply:
Added

Reply:
> I agree it doesn't make sense, but having this file is a requirement for ops level

:slightly_frowning_face:  Is this something we want to do/encourage? It feels a bit careless

> Can we add a comment that this does nothing and here to satisfy OpsLevel?

Could we remove everything in the file _except_ that comment?

>  don't see any "this repo doesn't need this file" option

Do other repos have this option? At a glance of the filter (here: https://app.opslevel.com/filters/4153) it looks like there's an (undocumented) it will filter on the tag `"app.uw.systems/tags.skip-repo"`.